index.d.ts

```diff
@@ -1195,9 +1196,10 @@ declare namespace Moleculer {
 		deserialize(type: string, data: Buffer): Packet;
 	}
 
+	type CacherKeygenFuncType<P = Record<string, any>, M = any, K = any[]|null> = (actionName: string, params: P, meta: M, defaultKeys: K) => string;
```

Comment:
The `Type` suffix here is probably extraneous.  We get the same benefit from naming this `CacherKeygenFunc` since we already know its a type.

Reply:
My personal preference is to use `unknown` in place of `any` whenever any generic is getting defaulted since `any` becomes a free-for-all without any type safety.  If a type has not been provided explicitly in a generic or inferred by TS, treating it as `any` becomes risky.